mod-finance/schemas/fiscal_year.json

```diff
@@ -14,7 +14,7 @@
     },
     "code": {
       "description": "The numeric code of the fiscal year",
-      "type": "integer"
+      "type": "string"
```

Comment:
should this description be updated too?

Reply:
Updated the description to reflect the type change